README.md

```diff
@@ -121,81 +95,71 @@ Also in the output will be the endpoints that will be used to interface with the
+
+`./ipld-eth-indexer backfill --config=<the name of your config file.toml>`
+
+* Resync: Manually define block ranges within which to (re)fill data over HTTP; can be ran in parallel with non-overlapping regions to scale historical data processing
+
+`./ipld-eth-indexer resync --config=<the name of your config file.toml>`
 
-`./ipfs-blockchain-watcher watch --config=<the name of your config file.toml>`
 
 ### Configuration
 
-Below is the set of universal config parameters for the ipfs-blockchain-watcher command, in .toml form, with the respective environmental variables commented to the side.
-This set of parameters needs to be set no matter the chain type.
+Below is the set of parameters for the ipld-eth-indexer command, in .toml form, with the respective environmental variables commented to the side.
+The corresponding CLI flags can be found with the `./ipld-eth-indexer {command} --help` command.
 
 ```toml
 [database]
     name     = "vulcanize_public" # $DATABASE_NAME
     hostname = "localhost" # $DATABASE_HOSTNAME
     port     = 5432 # $DATABASE_PORT
-    user     = "vdbm" # $DATABASE_USER
+    user     = "postgres" # $DATABASE_USER
     password = "" # $DATABASE_PASSWORD
 
-[watcher]
-    chain = "bitcoin" # $SUPERNODE_CHAIN
-    server = true # $SUPERNODE_SERVER
-    ipcPath = "~/.vulcanize/vulcanize.ipc" # $SUPERNODE_IPC_PATH
-    wsPath = "127.0.0.1:8082" # $SUPERNODE_WS_PATH
-    httpPath = "127.0.0.1:8083" # $SUPERNODE_HTTP_PATH
-    sync = true # $SUPERNODE_SYNC
-    workers = 1 # $SUPERNODE_WORKERS
-    backFill = true # $SUPERNODE_BACKFILL
-    frequency = 45 # $SUPERNODE_FREQUENCY
-    batchSize = 1 # $SUPERNODE_BATCH_SIZE
-    batchNumber = 50 # $SUPERNODE_BATCH_NUMBER
-    timeout = 300 # $HTTP_TIMEOUT
-    validationLevel = 1 # $SUPERNODE_VALIDATION_LEVEL
-```
+[log]
+    level = "info" # $LOGRUS_LEVEL
 
-Additional parameters need to be set depending on the specific chain.
+[sync]
+    workers = 4 # $SYNC_WORKERS
 
-For Bitcoin:
-
-```toml
-[bitcoin]
-    wsPath  = "127.0.0.1:8332" # $BTC_WS_PATH
-    httpPath = "127.0.0.1:8332" # $BTC_HTTP_PATH
-    pass = "password" # $BTC_NODE_PASSWORD
-    user = "username" # $BTC_NODE_USER
-    nodeID = "ocd0" # $BTC_NODE_ID
-    clientName = "Omnicore" # $BTC_CLIENT_NAME
-    genesisBlock = "000000000019d6689c085ae165831e934ff763ae46a2a6c172b3f1b60a8ce26f" # $BTC_GENESIS_BLOCK
-    networkID = "0xD9B4BEF9" # $BTC_NETWORK_ID
-```
-
-For Ethereum:
+[backfill]
+    frequency = 15 # $BACKFILL_FREQUENCY
+    batchSize = 2 # $BACKFILL_BATCH_SIZE
+    workers = 4 # $BACKFILL_WORKERS
+    timeout = 300 # $HTTP_TIMEOUT
+    validationLevel = 1 # $BACKFILL_VALIDATION_LEVEL
+
+[resync]
+    type = "full" # $RESYNC_TYPE
+    start = 0 # $RESYNC_START
+    stop = 0 # $RESYNC_STOP
+    batchSize = 2 # $RESYNC_BATCH_SIZE
+    workers = 4 # $RESYNC_WORKERS
+    timeout = 300 # $HTTP_TIMEOUT
+    clearOldCache = false # $RESYNC_CLEAR_OLD_CACHE
+    resetValidation = false # $RESYNC_RESET_VALIDATION
```

Comment:
Don't we want to organize it something like it https://github.com/helm/charts/tree/master/stable/prometheus-operator#general for example? In table format with some description and default value

Reply:
Yeah that would be an improvement, I think this should go into a separate PR/issue as this one is already very busy.